### PR TITLE
feat: article view title and cover

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://app.clickup.com/t/8678v0jt6

Note: the ticket doesnt mention the article wrapper but im adding it on this ticket, that should include the background with the patterns but I dont have that image yet. 

If i get the image before merging this pr will add it, otherwise can be handled as a separate task

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
I'll await the background image on this PR, as the border seems too close to the header content right now

File: resources/js/Pages/Articles/Show.tsx [hunk 1]
```diff
@@ -1,8 +1,25 @@
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
 import { DefaultLayout } from "@/Layouts/DefaultLayout";
 
 const ArticlesShow = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => (
     <DefaultLayout>
-        <div className="mx-6 sm:mx-8 2xl:mx-0">{article.title}</div>
+        <div className="mx-8 -mt-6 pb-6 pt-8 sm:-mt-8 md:px-24 lg:mt-0 lg:overflow-hidden lg:rounded-xl lg:border lg:border-theme-secondary-300">
```
Comment (ItsANameToo) [inline resources/js/Pages/Articles/Show.tsx:7]: can you double check the padding as it feels too close to the image compared with the design

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Comment by alfonsobries** [comment]
@ItsANameToo ✅ added background

File: resources/js/Pages/Articles/Show.tsx [hunk 2]
```diff
@@ -1,8 +1,27 @@
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
 import { DefaultLayout } from "@/Layouts/DefaultLayout";
 
 const ArticlesShow = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => (
     <DefaultLayout>
-        <div className="mx-6 sm:mx-8 2xl:mx-0">{article.title}</div>
+        <article className="article-wrapper -mt-6 pb-6 sm:-mt-8  lg:mx-8 lg:mt-0 lg:overflow-hidden lg:rounded-xl lg:border lg:border-theme-secondary-300">
+            <div className="bg-white/20 px-8 pt-8 md:px-24">
```
Comment (ItsANameToo) [inline resources/js/Pages/Articles/Show.tsx:8]: @alfonsobries this still has an issue with padding. Design is 1376px wide, while the implementation has padding which brings it down to 1310px.

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/js/Pages/Articles/Show.tsx (hunk 2 above)
Comment (ItsANameToo) [reply resources/js/Pages/Articles/Show.tsx:8]: the content should also be max 1000px wide on desktop

File: resources/css/_article.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@layer components {
+    .article-wrapper {
+        @apply bg-[url(../images/articles/article-wrapper-bg.svg)] bg-cover bg-top bg-no-repeat dark:bg-[url(../images/articles/article-wrapper-bg-dark.svg)];
```
Comment (ItsANameToo) [inline resources/css/_article.css:3]: the background image seems to span more height than it does in the design

**Comment by alfonsobries** [comment]
@ItsANameToo @shahin-hq paddng and bg fixed ✅

**Pull request merged**